src/control/lib/txtfmt/entity.go

```diff
@@ -0,0 +1,87 @@
+//
+// (C) Copyright 2019-2020 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package txtfmt
+
+import (
+	"fmt"
+	"text/tabwriter"
+)
+
+// EntityFormatter is a variant of TableFormatter that can be used
+// for neatly displaying attributes of a single entity.
+type EntityFormatter struct {
```

Comment:
Ah, but this is not OOP... EntityFormatter isn't a child -- it just embeds TableFormatter to benefit from commonalities. I'd only be inclined to add a baseTableFormatter (unexported to keep it within the package boundary) if we start adding a bunch of other table-based formatter types.

With regard to a Formatter interface -- IMO it's premature to define that. Generally interfaces should be defined by the consumer rather than the provider. If we had some function that accepted a generic formatter then it might make sense.